keyboards/reversestudio/decadepad/info.json

```
@@ -0,0 +1,38 @@
+{
+    "keyboard_name": "DecadePad",
+    "url": "",
+    "maintainer": "huajijam",
+    "width": 6,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_numpad_6x4": {
+            "key_count": 21,
+            "layout": [
+                {"x":0, "y":0},
+                {"x":0, "y":1},
+                {"x":0, "y":2},
+                {"x":0, "y":3},
+                {"x":0, "y":4},
+                {"x":0, "y":5, "w":2},
+
+                {"x":1, "y":0},
+                {"x":1, "y":1},
+                {"x":1, "y":2},
+                {"x":1, "y":3},
+                {"x":1, "y":4},
+
+                {"x":2, "y":0},
+                {"x":2, "y":1},
+                {"x":2, "y":2},
+                {"x":2, "y":3},
+                {"x":2, "y":4},
+                {"x":2, "y":5},
+
+                {"x":3, "y":0},
+                {"x":3, "y":1},
+                {"x":3, "y":2, "h":2},
+                {"x":3, "y":4, "h":2},
```

These objects need to be in the same order that `keymap.c` and `decadepad.h` use.

```suggestion
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},

                {"x":0, "y":2},
                {"x":1, "y":2},
                {"x":2, "y":2},

                {"x":0, "y":3},
                {"x":1, "y":3},
                {"x":2, "y":3},
                {"x":3, "y":2, "h":2},

                {"x":0, "y":4},
                {"x":1, "y":4},
                {"x":2, "y":4},

                {"x":0, "y":5, "w":2},
                {"x":2, "y":5},
                {"x":3, "y":4, "h":2}
```